### PR TITLE
add daniel bitengo's card

### PR DESCRIPTION
**Pull request by Dancode-188**
-Add Twitter handle and link
-Add GitHub handle and link
-Add bio section briefing about me
-Add 3 useful dev resources

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -282,6 +282,33 @@ <h2>A project for first-time contributions</h2>
...
+            <a href="https://x.com/DanielBitengo" target="_blank">@DanielBitengo</a>
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/Dancode-188" target="_blank">Dancode-188</a>
+          </p>
+          <p class="about">An aspiring software engineer with a passion for inovation. I like to work on embedded systems too</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="First Resource">LeetCode</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:299]:
```suggestion
                <a href="https://leetcode.com/" target="_blank" title="LeetCode">LeetCode</a>
```

File: index.html [hunk 2]
```diff
@@ -282,6 +282,33 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Second Resource">Dev.to</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:302]:
```suggestion
                <a href="https://dev.to/" target="_blank" title="Dev.to">Dev.to</a>
```

File: index.html [hunk 3]
```diff
@@ -282,6 +282,33 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Third resource">Medium</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:305]:
```suggestion
                <a href="https://medium.com/" target="_blank" title="Medium">Medium</a>
```

**Review by StackOverflowIsBetterThanAnyAI — COMMENTED** [review]
Please add valid href destinations and title attributes.

**Comment by ibrahimdevop21** [comment]
stall for more than 5 day's

**Pull request closed without merging**